setup.cfg

```diff
@@ -24,6 +24,7 @@ test =
     pytest
     pytest-doctestplus
     pytest-cov
+    pandas
```

Comment:
Pandas becomes a dependency in the testing suite to manage some of the tabular data records, but is not required to install or run PIPE (only the tests).